PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,15 @@
-# RFI Checklist
+## Newbies
 
-_**Before** submitting the pull request, fill out sections (1.) Related Issues and (2.) Pull Request Checklist._
+Regular contributors know the drill, but new contributors should at least glance at [CONTRIBUTING.md](/CONTRIBUTING.md). Please note that we measure code coverage and we'd love it if you write unit tests, but we're happy to review your pull request even if you don't! Thanks!
```

Comment:
Add something like "especially the parts that explain how we're happy to talk before you start" after asking them to read contributing.md